### DISCUSSION TITLE
turn on air conditioner my toyota

### DISCUSSION THREAD

**Discussion by SaracinoM** [DurgNomis-drol/ha_toyota#305] [Q&A]
HI guys
thanks for the job...do you know if we can have the sensor to turn on the air conditoner of the car on HA like on the app MYTOYOTA??
THANKS

**Comment by JamLaMin** [comment]
Is there any status update about this?

**Reply by CM000n** [reply]
https://github.com/DurgNomis-drol/ha_toyota/issues/346
